ci/k6-ingestion-test.js

```diff
@@ -1,6 +1,18 @@
 import http from 'k6/http'
-import {check, group, sleep} from 'k6'
+import {check, sleep, fail} from 'k6'
 import { Gauge } from 'k6/metrics'
+import { URL } from 'https://jslib.k6.io/url/1.0.0/index.js';
+
+const POSTHOG_ENDPOINT = __ENV.POSTHOG_ENDPOINT
+const POSTHOG_API_ENDPOINT = __ENV.POSTHOG_API_ENDPOINT || POSTHOG_ENDPOINT
+const POSTHOG_EVENT_ENDPOINT = __ENV.POSTHOG_EVENT_ENDPOINT || POSTHOG_ENDPOINT
```

Comment:
I don't think the extra configuration here is warranted (YAGNI), or am I missing something?